Notice.md

```diff
@@ -6,50 +6,78 @@ the open-source TEE maintained by Linaro, with initial contributions from
 STMicroelectronics, Ericsson and Linaro Limited.
 
 What OP-TEE is
-------
 
-OP-TEE is designed primarily to rely on the ARM TrustZone(R) technology as the
-underlying hardware isolation mechanism. However, it has been structured to be
-compatible with any isolation technology suitable for the TEE concept and goals,
-such as running as a virtual machine or on a dedicated CPU.
+OP-TEE is a Trusted Execution Environment designed as companion to a non secure
+Linux kernel running on ARM&reg; Cortex-A cores using the TrustZone&reg;
```

Comment:
Non-secure OS is not limited to Linux as you mention below so I suggest `s/Linux kernel/OS/`